src/Plugins/DBFTPlugin/Consensus/ConsensusService.cs

```diff
@@ -89,14 +99,17 @@ private void InitializeConsensus(byte viewNumber)
             {
                 if (isRecovering)
                 {
-                    ChangeTimer(TimeSpan.FromMilliseconds(neoSystem.Settings.MillisecondsPerBlock << (viewNumber + 1)));
+                    TimeSpan blockTime = GetBlockTimeFromPolicyContract();
```

Comment:
No, it should get `MillisecondsPerBlock` from neoSystem.Settings before `HF_Domovoi`.